parts/linux/cloud-init/config.ign

```diff
@@ -0,0 +1,8 @@
+{
+    "TODO": "POPULATE_ME",
```

Comment:
if we template as cloud-config and then convert to ignition, that works fine too. I didn't know how to easily generate ignition from cloud-config yaml without shelling to a binary

Reply:
his is really where the magic sauce should be